tests/e2e/utils/fetch-page-content.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ *
+ */
+import fetch from 'node-fetch';
+
+/**
+ * Helper to prepare the cookies to be passed to node-fetch
+ *
+ * @param {Array} cookies Array of cookies returned from page.getCookes().
+ *
+ * @return {string} The cookie string.
+ */
+export function prepareCookiesForHeader( cookies ) {
+	let parsedCookies = '';
+	cookies.forEach( ( cookie ) => {
+		parsedCookies += `${ cookie.name }=${ cookie.value }; `;
+	} );
+	return parsedCookies;
```

Comment:
Maybe not worth an extra utility for this as it's used only once anyways?
```suggestion
	return cookies.map( ( { name, value } ) => `${ name }=${ value };` ).join( ' ' );
```

Reply:
All of these have been updated